UPGRADE-API-1.12.md

```diff
@@ -225,3 +225,176 @@ The list of changed contexts:
 - Sylius\Behat\Context\Api\Shop\PromotionContext
 - Sylius\Behat\Context\Api\Shop\RegistrationContext
 - Sylius\Behat\Context\Api\Shop\ShipmentContext
+
+### Creating Request refactored
+
+We removed some methods from `Sylius/Behat/Client/RequestInterface`.
+
+ ```diff
+ -    public static function index(
+ -      ?string $section,
+ -       string $resource,
+ -       string $authorizationHeader,
+ -       ?string $token = null
+ -   ): self;
+
+ -   public static function subResourceIndex(?string $section, string $resource, string $id, string $subResource): self;
+
+ -   public static function show(
+ -       ?string $section,
+ -       string $resource,
+ -       string $id,
+ -       string $authorizationHeader,
+ -       ?string $token = null
+ -   ): self;
+
+ -   public static function create(
+ -       ?string $section,
+ -       string $resource,
+ -       string $authorizationHeader,
+ -       ?string $token = null
+ -   ): self;
+
+ -   public static function update(
+ -       ?string $section,
+ -       string $resource,
+ -       string $id,
+ -       string $authorizationHeader,
+ -       ?string $token = null
+ -   ): self;
+
+ -   public static function delete(
+ -       ?string $section,
+ -       string $resource,
+ -       string $id,
+ -       string $authorizationHeader,
+ -       ?string $token = null
+ -   ): self;
+
+ -   public static function transition(?string $section, string $resource, string $id, string $transition): self;
+
+ -   public static function customItemAction(?string $section, string $resource, string $id, string $type, string $action): self;
+
+ -   public static function upload(
+ -       ?string $section,
+ -       string $resource,
+ -       string $authorizationHeader,
+ -       ?string $token = null
+ -   ): self;
+
+ -   public static function custom(string $url, string $method, array $additionalHeaders = [], ?string $token = null): self;
+ ```
+
+All of them have been moved to new `Sylius/Behat/Client/RequestFactory` class.
```

Comment:
It's not literally true as these methods are no more static.
```suggestion
All of them have their corresponding implementation in `Sylius/Behat/Client/RequestFactory`.
```
or something like that and I would be for moving this sentence right behind:
>  We removed some methods from `Sylius/Behat/Client/RequestInterface`.